config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -57,6 +57,7 @@
   <!-- this file cannot be line wraped -->
   <suppress id="lineLength" files="config[\\/]signatures.txt"/>
   <suppress id="lineLength" files="config[\\/]signatures-test.txt"/>
+  <suppress id="lineLength" files="config[\\/]checkstyle-checks.xml"/>
```

Comment:
Suppressing the whole file is too harsh because of one line. I couldn't find a suppression that can be used. Maybe we can crate one with `SuppressWithNearbyTextFilter` that can be used like in the java code `-@cs[LineLength] <message>`?

Reply:
https://github.com/checkstyle/checkstyle/pull/13129/files#r1213607118 should address this

Reply:
lets use https://checkstyle.org/config_filters.html#SuppressWithNearbyTextFilter